spec/signin/Checklist_Sidebar.txt

```diff
@@ -0,0 +1,21 @@
+Sidebar
+1. Verify that 'Tops' filter tab is opens
+2. Verify that 'Dresses' filter tab is open
+3. Verify that 'Tops' filter tab select
+4. Verify that 'Dresses' tab filter select
+5. Verify that 'Size' filter tab is select
+6. Verify that 'Color' filter tab is select
+7. Verify that 'Compositions' filter tab is select
+8. Verify that 'Styles' filter tab is select
+9. Verify that 'Properties' filter tab is select
+10. Verify that 'Availability' filter tab is select
+11. Verify that 'Manufacturer' filter tab is select
+12. Verify that 'Condition' filter tab is select
```

Comment:
Sorry. Don't understand what do these checklists mean?
5. Verify that 'Size' filter tab is select.  ---- should be
Verify Catalog page contains available filters for items displayed: Size, Color, Composition....
Verify Catalog page filter Size displays available size option to filter: S, L, M
Verify selecting Size filter option M displays only items with size M on page
Also can be checklist for case, when page is displaying only one item, so the filters must adjust and display only one option.